src/components/pageTemplates/NodeTprUnitPage.tsx

```diff
@@ -113,23 +149,46 @@ function NodeTprUnitPage({ node, sidebar, ...props }: NodeTprUnitProps): JSX.Ele
             )}
 
             <ContactInfo />
-            
+
             {field_content?.length > 0 && (
-              <ContentMapper content={field_content} pageType='tpr_unit' locationId={drupal_internal__id} mapId={service_map_embed} />
+              <ContentMapper
+                content={field_content}
+                pageType='tpr_unit'
+                locationId={drupal_internal__id}
+                mapId={service_map_embed}
+              />
             )}
           </div>
-          <div className="sidebar col col-4 flex-order-first">
-            <Sidebar {...sidebar}/>
+          <div className='sidebar col col-4 flex-order-first'>
+            <Sidebar {...sidebar} />
             <ContactInfo aside={true} />
           </div>
         </div>
-        <div className="columns">
-          <div className="lower-content-region col col-12">
+        <div className='columns'>
+          <div className='lower-content-region col col-12'>
             {field_lower_content?.length > 0 && (
-              <ContentMapper content={field_lower_content} pageType='tpr_unit' />
+              <ContentMapper
+                content={field_lower_content}
+                pageType='tpr_unit'
+              />
             )}
           </div>
         </div>
+        {node.accessibility_sentences.length !== 0 && (
+          <AccordionWithIcon
```

Comment:
Let's move the groups array formatting here so it's outside the accordion.

Reply:
Array been moved outside the component